src/components/Page.jsx

```diff
@@ -7,6 +7,12 @@ import "./../index.css";
 const Page = () => {
  
   const [activeStep, setActiveStep] = useState(0);
+  const [usernameError, setUsernameError] = useState(false);
```

Comment:
Instead of tracking each error individually in state, can you just create an error object in state?
```
const error = {
  name: false,
  email: false,
  phone: false,
}
```